docs/source/virtual_ecosystem/microclimate_theory.md

```diff
@@ -0,0 +1,288 @@
+* **Surface temperature**: The temperature of the ground or other surfaces can be
+significantly different from the air temperature above; often it is much hotter.
+
+### Humidity
+
+* **Relative humidity**: The percentage of moisture in the air relative to the maximum
+amount the air can hold at that temperature. It impacts endotherm comfort, plant
+transpiration, and microbial activity.
+* **Vapour pressure deficit (VPD)**: The difference between the amount of
+moisture in the air and how much moisture the air can hold when it is saturated. VPD is a
+crucial factor for plant transpiration and water stress; higher VPD indicates a greater
+potential for evaporation and transpiration.
+* **Soil moisture**: The amount of water contained within soil affects evaporation rates
+, soil microbial activity, and the availability of water for plants.
+* **Soil matric potential**: Soil matric potential refers to the energy status of water
+in soil which represents the force per unit area that the soil matrix exerts on water due
+to capillary and adsorptive forces. It influences the movement and availability of water
+to plants, with lower values indicating drier conditions and higher values indicating
+wetter conditions.
+
+### Wind
+
+* **Wind Speed**: The rate at which air is moving horizontally influences
+temperature regulation, evaporation rates, and the dispersion of atmospheric gasses.
+* **Wind Direction**: The direction from which the wind is blowing affects microclimate
+by influencing temperature distribution, humidity, and the transport of seeds and pollen.
+* **Turbulence**: Turbulence refers to the irregular, chaotic movement of air that affects
+the mixing of atmospheric elements. Turbulence influences heat exchange, moisture
+distribution, and the dispersion of aerosols.
+
+### Precipitation
+
+* **Rainfall**: The amount of rain that falls in a given area over a specific period.
+Rainfall is vital for replenishing water sources and maintaining soil moisture.
+* **Snowfall**: The amount of snow that falls, which can be measured as snow depth or water
+equivalent. Snowfall influences temperature regulation, soil moisture, and water supply.
+* **Dew and Frost**: The formation of water droplets or ice crystals on surfaces, which can
+affect plant health and soil conditions. Dew and frost are critical in areas where
+precipitation is low.
+
+### Atmospheric Pressure
+
+Barometric pressure describes the pressure exerted by the atmosphere at a given point.
+Pressure decreases with height and influences weather patterns; changes in atmospheric
+pressure can indicate upcoming weather changes, such as storms.
+
+### Atmospheric $\ce{CO_{2}}$
+
+Although atmospheric $\ce{CO_{2}}$ is not a microclimatic variable, it is closely tied
+in the balance of energy through its critical role in photosynthesis. It should therefore
+be considered when studying the dynamics on ecosystems.
+
+## Links to biotic components
```

Comment:
I feel like this section could maybe do with a (small) introduction, as it feels a bit of an abrupt change after the "key microclimate variables" section